### PR TITLE
enhance : Contribute Guideline

### PR DESCRIPTION
**Pull request by bipon68**
Contribute Guideline

### CODE REVIEW

File: contribute.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+### Contribute Guideline
+Generally speaking, you need to go through the following steps to become a contributor:
+
+- Fork a repo
+- Clone the repo
+- Define a pre-commit hook
```
Comment (umarfchy) [inline contribute.md]:
```suggestion
```

Comment (umarfchy) [reply contribute.md]: Remove this. Not required.

**Review by umarfchy — CHANGES_REQUESTED** [review]

File: contribute.md (hunk 1 above)
Comment (bipon68) [reply contribute.md]: Removed: Define a pre-commit hook this point. Thanks for your suggestion.

**Review by umarfchy — APPROVED** [review]
LGTM!

**Pull request merged**